docker-compose/mariadb-config/prod/mariadb.cnf

```diff
@@ -0,0 +1,58 @@
+[mariadb]
+
+port                  = 3306
+bind-address          = 0.0.0.0
+
+# Fine tuning
+max_connections         = 300
+connect_timeout         = 5
+wait_timeout            = 600
+max_allowed_packet      = 32M
+thread_cache_size       = 128
+sort_buffer_size        = 128M
+bulk_insert_buffer_size = 128M
+tmp_table_size          = 32M
+max_heap_table_size     = 32M
+skip-name-resolve       = 1
+innodb_log_file_size    = 2G
+
+# Logging
+# warning log-basename is defined later for replication
+# some log option may be ignored.
+# see: https://mariadb.com/kb/en/mysqld-options/#-log-basename
+# log_error = /var/log/mysql/error.log
+slow_query_log
+slow_query_log_file    = /var/log/mysql/mariadb-slow.log
+long_query_time        = 2
+log_slow_verbosity     = query_plan,explain
+log_error              = /var/log/mysql/mariadb.err.log
+
+# Query cache
+query_cache_size        = 16M
+
+# Character sets
+character-set-server = utf8mb4
+collation-server     = utf8mb4_general_ci
```

Comment:
utf8mb4_general_ci is a horrible default, utf8mb4_uca1400_ai_ci would be better. It would solve PRs that come in with branches/description in modern collations.

*HOWEVER* to do ideally all bb tables that have a collation would been to be changed and therefore indexes rebuild (otherwise JOINs may not use index). And testing. Its a big move that might be required eventually. So don't change until you're ready. Don't leave too long or the tables will get big and slow to alter beyond an acceptable outage window.

Reply:
Thank you for the advice @grooverdan ,
yes, it's preferable to do this after the migration, starting with some full-runs on the Development environment.